### ISSUE TITLE
Contrast for #wdn_menu_toggle

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#700]
WCAG2AA
Discovered with: http://squizlabs.github.io/HTML_CodeSniffer/

```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.27:1. Recommendation: change background to #a40000.
```

Code Snippet:

```
<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
```
